package.json

```diff
@@ -20,16 +20,20 @@
     "standard-http-error": "^2.0.1"
   },
   "devDependencies": {
-    "eslint": "^3.16.1",
-    "eslint-config-seegno": "^9.0.0",
-    "jest": "^19.0.2",
+    "eslint": "^6.5.1",
+    "eslint-config-uphold": "^1.0.1",
+    "jest": "^24.9.0",
     "koa": "^2.0.1",
     "lint-staged": "^3.4.0",
     "pre-commit": "^1.2.2",
+    "prettier": "^1.18.2",
     "supertest": "^3.0.0"
   },
   "engines": {
-    "node": ">=7.6"
+    "node": "^8.10.0 || ^10.13.0 || >=11.10.1"
```

Comment:
Simplify? The rules for eslint shouldn't have to matter to projects that depend on this one.